client/src/main/java/org/spine3/validate/Validate.java

```diff
@@ -103,7 +104,7 @@ public static boolean isNotDefault(Message object) {
      */
     public static <M extends Message> M checkNotDefault(M object) {
         checkNotNull(object);
-        checkNotDefault(object, "The message is in the default state: %s", object);
+        checkNotDefault(object, "The message is in the default state: %s", TypeName.of(object).value());
```

Comment:
Having `toString()` in `TypeName` makes the `value()` call redundant. I think we can make the code more terse here.